<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecrodDB</class>
 <widget class="QDialog" name="RecrodDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecrodDB</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QPushButton{
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
padding:6px;
}*/</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>211</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
font: 11pt &quot;3ds&quot;;
}</string>
   </property>
   <property name="title">
    <string>当前记录信息</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前动作描述信息：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="RecDiscrption">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>实现任务，注意事项</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>188</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>记录序号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="RecNo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>写入日期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>动作数量</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="ActionCount">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>22</y>
     <width>100</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnOpenDB">
      <property name="text">
       <string>打开数据库</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCloseDB">
      <property name="text">
       <string>关闭数据库</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnInsert">
      <property name="text">
       <string>插入当前记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRead">
      <property name="text">
       <string>读取选中记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>删除选中记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReturn">
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>391</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="actRecFirst">
      <property name="text">
       <string>首记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actRecPrevious">
      <property name="text">
       <string>前一条记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actRecNext">
      <property name="text">
       <string>后一条记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actRecLast">
      <property name="text">
       <string>尾记录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>RecrodDB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
